src/views/AddLocusGeneDiseaseView.vue

```diff
@@ -435,6 +479,7 @@ export default {
         </button>
         <button
           v-else
+          :disabled="isGeneDataLoading"
           type="button"
           class="btn btn-primary mb-3"
           @click="geneSearchBtnClickHandler"
```

Comment:
Suggesting that we also use `@keyup-enter` to be able to use enter when a gene is entered. 
The `@keyup.enter="geneSearchBtnClickHandler"` needs to be added to L466

Reply:
taken care